extensions/wikia/TrackingOptIn/dist/tracking-opt-in.min.js

```diff
@@ -5,4 +5,4 @@ define("wikia.trackingOptInModal",[],function(){return function(e){var n={};func
chent pour vous et que certaines fonctionnalités du site ne fonctionnent pas comme prévu. Si vous changez d'avis, vous pouvez revenir sur votre choix à tout moment en consultant notre politique de confidentialité.  Pour plus d'informations sur les cookies utilisés, voir notre Politique de confidentialité. Pour plus d'informations sur nos partenaires utilisant des cookies publicitaires sur le site, voir la Liste des partenaires ci-dessous."],buttonAccept:"ACCEPTER LES COOKIES PUBLICITAIRES",buttonReject:"REFUSER LES COOKIES PUBLICITAIRES",privacyLinkText:"Politique de confidentialité de FANDOM",privacyLink:"http://www.wikia.com/Privacy_Policy",partnerLinkText:"Liste des partenaires",partnerLink:"http://fandom.wikia.com/partner-list"},it:{headline:"Questo sito utilizza i cookie",bodyParagraphs:['FANDOM e i nostri partner usano tecnologie come i cookie sul nostro sito per personalizzare contenuti, annunci, fornire funzioni social e analizzare il nostro traffico. Vorremmo concentrare la tua attenzione in particolar modo sui cookie che sono usati a scopi commerciali o per fornire annunci personalizzati per te. Quando usiamo questi cookie ("cookie commerciali"), lo facciamo solo se hai acconsentito al loro uso selezionando "ACCETTA COOKIE COMMERCIALI" qui sotto. Considera che nel caso di rifiuto potresti vedere annunci meno pertinenti per te e certe funzioni del sito potrebbero non funzionare come previsto. Puoi cambiare idea e rivisitare le tue scelte di consenso in qualsiasi momento visitando la nostra informativa sulla privacy. Per ulteriori informazioni sui cookie che usiamo, si prega di consultare la nostra informativa sulla privacy. Per informazioni sui nostri partner che usano cookie per la pubblicità sul nostro sito, consulta per favore la lista dei partner qui sotto.'],buttonAccept:"ACCETTA COOKIE COMMERCIALI",buttonReject:"RIFIUTA COOKIE COMMERCIALI",privacyLinkText:"Informativa sulla privacy di FANDOM",privacyLink:"http://www.wikia.com/Privacy_Policy",partnerLinkText:"Lista dei partner",partnerLink:"http://fandom.wikia.com/partner-list"},ja:{headline:"このサイトではCookieを使用しています",bodyParagraphs:["FANDOMとそのパートナーはこのサイトでCookieなどの技術を使用し、コンテンツや広告をユーザーに合わせてカスタマイズしたり、ソーシャルメディア機能を提供したり、トラフィックを分析したりしています。 特に、ユーザーをターゲットにしたり、ユーザーにカスタマイズされた広告を表示したりするために使用されるCookieにはご注意ください。 FANDOMとパートナーがこうしたCookie（「広告Cookie」）を使用するのは、ユーザーが下の「広告Cookieを許可」をクリックして、FANDOMやパートナーがこうしたCookieを使用することに同意した場合だけです。 なお、拒否を選択した場合は、関連性の低い広告が表示される可能性や、このサイトの一部の機能が正常に動作しない可能性があることにご注意ください。 この選択は、個人情報保護方針のページにアクセスして、後からいつでも変更していただけます。  FANDOMとパートナーが使用するCookieについて詳しくは、「個人情報保護方針」をご覧ください。 このサイトで広告Cookieを使用しているFANDOMのパートナーについて詳しくは、下の「パートナー・リスト」をご覧ください。"],buttonAccept:"広告Cookieを許可",buttonReject:"広告Cookieを拒否",privacyLinkText:"FANDOM個人情報保護方針",privacyLink:"http://www.wikia.com/Privacy_Policy",partnerLinkText:"パートナー・リスト",partnerLink:"http://fandom.wikia.com/partner-list"},pl:{headline:"Ta witryna używa plików cookie",bodyParagraphs:['FANDOM i nasi partnerzy korzystają z technologii takich jak pliki cook
```

Comment:
Does it make sense to put all possible i18n strings in a single JS file?

Reply:
JS in 2018 in general makes no sense to me ;3